### ISSUE TITLE
Upload Timed Out. Unsure if deployment was successful or not.

### ISSUE THREAD

**Issue by tejaswankalluri** [Azure/static-web-apps#1396]
![image](https://github.com/Azure/static-web-apps/assets/49343044/5c3de4f6-cdc6-42d9-9b67-636593122830)

**Comment by FinOCE** [comment]
I'm experiencing the same issue, first occurred for me half an hour ago. I've attempted to deploy two separate SWA sites and both experiencing the issue. It looks to me like the deployments did not work as I don't see my changes reflected on the site. We'll probably just have to wait until its fixed before we can deploy anything again.

**Comment by pondchamp** [comment]
+1, this issue is blocking all SWA deployment pipelines on our end

**Comment by nerblock** [comment]
Same on our end, also the failed deployment broke the staging site.

**Comment by sonuame** [comment]
Same here for me

**Comment by takawww** [comment]
same here

**Comment by 58836980** [comment]
working now~

**Comment by youngox9** [comment]
same here

**Comment by tejaswankalluri** [comment]
> working now~

Yes It's working now

**Comment by takawww** [comment]
> working now~

Yes, is it working now, thanks for notice

**Comment by sonuame** [comment]
Yes its working again for me as well

**Comment by K3llr** [comment]
Same issue. Not working currently.

**Comment by joshuaolusayo** [comment]
I am also having the issue... The problem is that.. how will I know if the deployment is successful because the build seems to be failing with the error above.

The first time I checked my latest build, it was not successful. No changes on the server also. I re-run the job in the github actions and it was not successful again - the build failed. When I checked the website, the changes seems to have been uploaded... How do I know next time that my update has been successfully deployed if the changes made cannot really be easily seen on the website?

**Comment by owljackob** [comment]
+1, started recently.

**Comment by kristiangronli** [comment]
same issue here

**Comment by chenn88** [comment]
Having this issue also.

**Comment by bcampru** [comment]
We are also having this issue

**Comment by chenn88** [comment]
It seems that mine does build and deploy (eventually) on SWA, despite the timeout from GH workflow. When I checked environments on SWA it said uploading for a while then changed to ready, but the GH workflow still had it recorded as incomplete.

**Comment by phoenixz0024** [comment]
same for us

**Comment by tuomasorhanen** [comment]
Same issue here!

**Comment by boriseetgerink** [comment]
I'm also affected by this issue. Deployments worked fine without functions, but started to timeout when I added functions to my site.

**Comment by compinschools** [comment]
Getting the same issue here

**Comment by rduque1** [comment]
Same issue here and when I build the app and check the size it 's only 51MB

**Comment by firasnajjar** [comment]
same here

**Comment by egbertbeuker** [comment]
Same, Azure Static App deployment

**Comment by tsubakimoto** [comment]
same issue here

**Comment by chenn88** [comment]
> It seems that mine does build and deploy (eventually) on SWA, despite the timeout from GH workflow. When I checked environments on SWA it said uploading for a while then changed to ready, but the GH workflow still had it recorded as incomplete. 

This is no longer working for me at all. Now just taking about 30minuted to say failed in the SWA environment.

**Comment by JorisDePoris** [comment]
Same issue here

**Comment by jpaholik** [comment]
Since today I have the same issue

**Comment by grantvine-sc** [comment]
Also have this issue, any updates ?

**Comment by krisnashaypp** [comment]
+1

**Comment by GojkoGalonja** [comment]
Same issue

**Comment by Freshchris01** [comment]
Same issue here. Did you open a support ticket to raise awareness?

**Comment by jesseflikweerteo** [comment]
@egbertbeuker, it's likely not a GitHub issue as it's present on Azure DevOps as well.

**Comment by egbertbeuker** [comment]
@jesseflikweerteo thank's, do you know if this issue has been reported to Microsoft and where I can find its status?

**Comment by jesseflikweerteo** [comment]
@egbertbeuker, no, I don't.

**Comment by blackspike** [comment]
same

**Comment by Freshchris01** [comment]
I tried to deploy my React frontend manually with the [swa cli](https://azure.github.io/static-web-apps-cli/docs/cli/swa-deploy) and experience the issue without Github Actions as well.

From my experience opening support tickets and including a link to the issue will help raise awareness.

**Comment by kornhammer** [comment]
+ same issue here as well

**Comment by christie-hill-za** [comment]
I have also been experiencing this issue for the past two hours trying to deploy a Nuxt Static Web App to Azure. No build errors, just times out after about 6-9 minutes 

![image](https://github.com/Azure/static-web-apps/assets/83393649/8d9715db-7088-44c0-9463-c53376bb3c41)

**Comment by ulfdev** [comment]
Same here, just started to happen today.

**Comment by rjoshi** [comment]
Facing same issue since yesterday evening, on Azure protal I noticed the deployment eventually happens but the github action is not passing.


---End of Oryx build logs---
Try to validate location at: '/bin/staticsites/591fd2a1-1e5f-4550-89ac-fbe2fcac995c-swa-oryx/app/.next'.
Oryx has determined that the app build artifacts are in the '.next' directory. If this is incorrect please specify the app artifact location.
Finished building app with Oryx
Detected standalone folder, so using it for deployment.
Repackaged Next.js app to deploy on the backend.
Skipping step to build /bin/staticsites/591fd2a1-1e5f-4550-89ac-fbe2fcac995c-swa-oryx/app/.next/standalone with Oryx
Function Runtime Information. OS: linux, Functions Runtime: ~4, node version: 18
Zipping Api Artifacts
Done Zipping Api Artifacts
Zipping App Artifacts
Done Zipping App Artifacts
Uploading build artifacts.
Finished Upload. Polling on deployment.
Status: InProgress. Time: 0.0888352(s)
Status: InProgress. Time: 15.169784(s)
Status: InProgress. Time: 30.2546991(s)
Status: InProgress. Time: 45.341156(s)
Status: InProgress. Time: 60.4209056(s)
Status: InProgress. Time: 75.5075716(s)
Status: InProgress. Time: 90.5889109(s)
Status: InProgress. Time: 105.665616(s)
Status: InProgress. Time: 120.7504045(s)
Status: InProgress. Time: 135.8308853(s)
Status: InProgress. Time: 150.9156537(s)
Status: InProgress. Time: 166.007786(s)
Status: InProgress. Time: 181.0956602(s)
Status: InProgress. Time: 196.1774999(s)
Status: InProgress. Time: 211.2725957(s)
Status: InProgress. Time: [226](https://github.com/StarSpark-AI/worksheets/actions/runs/7692546319/job/20974294487#step:4:227).351[232](https://github.com/StarSpark-AI/worksheets/actions/runs/7692546319/job/20974294487#step:4:233)3(s)
Upload Timed Out. Unsure if deployment was successful or not.

For further information, please visit the Azure Static Web Apps documentation at https://docs.microsoft.com/en-us/azure/static-web-apps/
If you believe this behavior is unexpected, please raise a GitHub issue at https://github.com/azure/static-web-apps/issues/
Exiting

**Comment by danbord** [comment]
same here

**Comment by eolasd** [comment]
same here, build runs ok, stuck uploading on swa..
![failswa](https://github.com/Azure/static-web-apps/assets/15805713/2847036a-649a-409f-8c45-497c4641aaac)

Github actions log shows build completing but timeout on polling the deploy to azure.

**Comment by mikeschepers4** [comment]
Same here

**Comment by bjoernAIO** [comment]
same for us

**Comment by owljackob** [comment]
https://github.com/Azure/static-web-apps/issues/1404#issuecomment-1915188240

**Comment by danielmellqvist** [comment]
Same issue here!

**Comment by aerickson-clt** [comment]
Same issue here

**Comment by 20-dishes** [comment]
Same issue, started earlier today

**Comment by JoelMTaylor** [comment]
Now, I cannot get it to succeed at all. Anyone else having it time out _always_?

**Comment by hugoforte** [comment]
Mine are always timing out as well

**Comment by chenn88** [comment]
> Now, I cannot get it to succeed at all. Anyone else having it time out _always_? 

Yeah tried several times today but timing out on every attempt. Looks like the Azure are aware and looking into it.

**Comment by asiz15** [comment]
:eyes:

**Comment by eolasd** [comment]
Commenting out API_LOCATION fixed issue for me (see #1404)
Deploy successful just now.

**Comment by aerickson-clt** [comment]
> Commenting out API_LOCATION fixed issue for me (see #1404) Deploy successful just now.

I can confirm this, but since my app has a managed API it is not a viable workaround.  I guess an alternative would be to use an unmanaged Azure Function, but I this is not practical considering that the issue will soon be fixed.

**Comment by javiercoh** [comment]
> Commenting out API_LOCATION fixed issue for me (see #1404) Deploy successful just now.

this also worked for me

**Comment by aerickson-clt** [comment]
According to the tracking at https://app.azure.com/h/2S4Z-JTG/6d979c the issue has been resolved, and my deployment (with api_location) is now working.

**Comment by GojkoGalonja** [comment]
I just ran my pipeline and it is deployed with no issues.

**Comment by joshuaolusayo** [comment]
It's working fine now from my end also...

**Comment by thomasgauvin** [comment]
This issue was resolved in January, closing. Please open a new issue if you encounter this in the future.

**Comment by sarveshrekhta** [comment]
I m getting same issue. Please suggest. Its critical.

**Comment by farazirfan47** [comment]
same issue

**Comment by carlohoanguyen** [comment]
Same issue here. Did anyone find a solution or the root problem?

**Comment by rduque1** [comment]
For me it was the middleware, the Azure health checks need to work in order to deploy successfully the application:
[https://learn.microsoft.com/en-us/azure/static-web-apps/deploy-nextjs-hybrid#configure-routing-and-middleware-for-deployment](https://learn.microsoft.com/en-us/azure/static-web-apps/deploy-nextjs-hybrid#configure-routing-and-middleware-for-deployment)

Maybe this helps

**Comment by carlohoanguyen** [comment]
Trying to deploy a minimal NextJS app to SWA with Identity token or Azure access token still times out.

Followed the docs on configuration: https://learn.microsoft.com/en-us/azure/static-web-apps/build-configuration?tabs=aat&pivots=github-actions

Next.js version: 15.4.6
